boards/arm/atsamd21_xpro/pinmux.c

```diff
@@ -82,6 +82,10 @@ static int board_pinmux_init(struct device *dev)
 #if (ATMEL_SAM0_DT_SERCOM_CHECK(5, atmel_sam0_i2c) && CONFIG_I2C_SAM0)
 #warning Pin mapping may not be configured
 #endif
+#if (ATMEL_SAM0_DT_TCC_CHECK(0, atmel_sam0_tcc_pwm) && CONFIG_PWM_SAM0_TCC)
```

Comment:
nit: add a blank line so the SERCOM related code is separated from the TCC related code

Reply:
Fixed